test/confounders.jl

```diff
@@ -56,7 +56,6 @@ end
     
     clean(parsed_args)
 
-    # No qc file provided
```

Comment:
why is that comment removed?

Reply:
Unsure, it may have happened when I was resolving merge conflicts. I can add this back in.

Reply:
added back in.